tests/FSharp.Data.GraphQL.IntegrationTests.Server/Startup.fs

```diff
@@ -6,14 +6,18 @@ open Microsoft.Extensions.DependencyInjection
 open Giraffe
 open Microsoft.Extensions.Logging
 open System
+open Microsoft.AspNetCore.Server.Kestrel.Core
 
 type Startup private () =
     new (configuration: IConfiguration) as this =
         Startup() then
         this.Configuration <- configuration
 
     member __.ConfigureServices(services: IServiceCollection) =
-        services.AddGiraffe() |> ignore
+        services.AddGiraffe()
+                .Configure(Action<KestrelServerOptions>(fun x -> x.AllowSynchronousIO <- true))
```

Comment:
We should probably track down the `Async.RunSynchronously` calls, but we can do that in a followup PR.

Reply:
Yes, I can work on it on my next free time. I had an issue where we are having an asynchronous error when executing queries on the server, and I could not figure it at the time, so I let it as it is for now. I will look at it this weekend.